### PR TITLE
Added more logging info for target roundtrip

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cedar-drt/fuzz/fuzz_targets/roundtrip.rs [hunk 1]
```diff
@@ -91,12 +91,16 @@ fn round_trip_ast(p: &StaticPolicy) -> StaticPolicy {
 fn round_trip_json(p: StaticPolicy) -> StaticPolicy {
     // convert to json
     let est = est::Policy::from(ast::Policy::from(p));
+    info!("round_trip_json est: {est:?}");
     let json = serde_json::to_value(est).expect("failed to convert EST to JSON");
+    info!("round_trip_json json: {}", json.to_string());
```
Comment (cdisselkoen) [inline cedar-drt/fuzz/fuzz_targets/roundtrip.rs:96]:
```suggestion
    info!("round_trip_json json: {json}");
```

File: cedar-drt/fuzz/fuzz_targets/roundtrip.rs [hunk 2]
```diff
@@ -112,24 +116,29 @@ fn round_trip_est(p: &StaticPolicy) -> StaticPolicy {
...
         )
     });
+    info!("round_trip_est est: {est:?}");
     let json = serde_json::to_value(est).unwrap_or_else(|err| {
         panic!(
             "Failed to convert EST to JSON: {:?}\nParse error: {:?}\n",
             p, err
         )
     });
+    info!("round_trip_est json: {}", json.to_string());
```
Comment (cdisselkoen) [inline cedar-drt/fuzz/fuzz_targets/roundtrip.rs:126]:
```suggestion
    info!("round_trip_est json: {json}");
```

**Review by cdisselkoen — COMMENTED** [review]
1) Will all of this logging slow down the fuzz execution any?  Or will the logging only happen during debugging, and not during real fuzzing runs?

2) Do we have a policy about what debug-logging we check in?  Other targets don't have this level of logging.  How do we decide whether to check-in this kind of code to `cedar-spec`, vs just add it locally as needed for debugging particular issues?  (I'm not saying we shouldn't take this PR, just wondering in general how we decide what logging we should and shouldn't include in our fuzz targets)

**Comment by shaobo-he-aws** [comment]
> 1. Will all of this logging slow down the fuzz execution any?  Or will the logging only happen during debugging, and not during real fuzzing runs?

I think it should not if we don't enable logging, which should only be used for stats collection and debugging.

> 2. Do we have a policy about what debug-logging we check in?  Other targets don't have this level of logging.  How do we decide whether to check-in this kind of code to `cedar-spec`, vs just add it locally as needed for debugging particular issues?  (I'm not saying we shouldn't take this PR, just wondering in general how we decide what logging we should and shouldn't include in our fuzz targets)

No, I think it's kind of ad-hoc and more like demand-driven.

**Comment by john-h-kastner-aws** [comment]
> > 1. Will all of this logging slow down the fuzz execution any?  Or will the logging only happen during debugging, and not during real fuzzing runs?
> 
> I think it should not if we don't enable logging, which should only be used for stats collection and debugging.

This should be configurable at run time with `RUST_LOG=debug` so that we can have fuzzing run with `RUST_LOG=error` to limit logging, but then we can re-run the failing unit with `RUST_LOG=debug` so that the logs show the more verbose output.

**Review by cdisselkoen — APPROVED** [review]
Personally I think it's fine to check these in, curious on others' thoughts though

**Review by khieta — APPROVED** [review]
Also seems fine to me to merge this. Just make sure that we are using the appropriate `RUST_LOG` level when we run DRT.

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by shaobo-he-aws** [comment]
Closed as the code has diverged too much from main.

**Pull request closed without merging**